repos/fdbt-site/tests/testData/mockData.ts

```diff
@@ -4358,7 +4459,7 @@ export const mockReturnValidityFieldset: RadioConditionalInputFieldset = {
                     id: 'return-validity-units',
                     name: 'duration',
                     label: 'Duration',
-                    options: ['day', 'week', 'month', 'year'],
+                    options: ['day', ExpiryUnit.WEEK, 'month', 'year'],
```

Comment:
Should we make all of these either 'day' or 'ExpiryUnit.day' format?

Reply:
Done 👍